helix-view/src/editor.rs

```diff
@@ -1715,3 +1715,106 @@ fn try_restore_indent(doc: &mut Document, view: &mut View) {
         doc.apply(&transaction, view.id);
     }
 }
+
+static EXPAND_ARGS_REGEXP: once_cell::sync::Lazy<helix_core::regex::Regex> =
+    once_cell::sync::Lazy::new(|| {
+        helix_core::regex::Regex::new(r"%(\w+)\{([^{}]*(\{[^{}]*\}[^{}]*)*)\}").unwrap()
+    });
+
+pub fn expand_args<'a>(editor: &Editor, args: &'a str) -> anyhow::Result<Cow<'a, str>> {
```

Comment:
```suggestion
pub fn expand_variables<'a>(editor: &Editor, input: &'a str) -> anyhow::Result<Cow<'a, str>> {
```

This will eventually be used for things other than arguments